dss/stepfunctions/gscopyclient/implementation.py

```diff
@@ -60,7 +60,7 @@ def run(self) -> dict:
             state = self.get_state_copy()
             src_blob = self.gcp_client.bucket(self.source_bucket).get_blob(self.source_key)
             dst_blob = self.gcp_client.bucket(self.destination_bucket).blob(self.destination_key)
-            content_type = src_blob._get_content_type(None)
+            content_type = src_blob.content_type
```

Comment:
Google client libraries do not automatically populate blob properties.

Please add `src_blob.reload()` before this line.

Reply:
Nevermind, @DailyDreaming  pointed out that `get_blob` [does reload for you](https://github.com/googleapis/google-cloud-python/blob/4166ddbf1f8b07c2a5e2db8f7955457b5cedd82d/storage/google/cloud/storage/bucket.py#L705)